test/ex_machina/ecto_test.exs

```diff
@@ -67,4 +82,17 @@ defmodule ExMachina.EctoTest do
       Factory.params_for(:user_map)
     end
   end
+
+  test "params_with_assocs/2 inserts associated fields" do
+    params = Factory.params_with_assocs(:article)
+
+    assert Map.keys(params) == [:author_id, :title]
+
+    %{
+      title: "My Awesome Article",
+      author_id: author_id,
+    } = params
+
+    assert author_id != nil
```

Comment:
I think this might be easier to visualize if we asserted against an exact map

```elixir
assert params == %{
  title: "My Awesome Article",
  author_id: TestRepo.one!(User).id
}
```